### PR TITLE
Added FUNAI to school list

### PR DESCRIPTION
**Pull request by ikemHood**

### CODE REVIEW

File: src/list.json [hunk 1]
```diff
@@ -140794,5 +140794,19 @@
...
+      "funai.edu.ng"
+    ],
+    "web_pages": [
+      "https://funai.edu.ng"
+    ],
+    "country": "Nigeria",
+    "alpha_two_code": "NG",
+    "state-province": null,
+    "id": 10033,
+    "abbr": "aefunai"
```
Comment (Luluno01) [inline src/list.json:140810]: Thanks for the PR! Would you please confirm if this is the correct abbreviation of your university? The abbreviation will be **_displayed as-is without converting to upper case letters_**.

**Comment by ikemHood** [comment]
Yes, that is the correct abbr, only that it is lowercase. 
Let me make them uppercase. 
The most i saw were lowercases so i thought that was a convention.

File: src/list.json (hunk 1 above)
Comment (ikemHood) [reply src/list.json:140810]: Yes, that is the correct abbr, only that it is lowercase.
Let me make them uppercase.
The most i saw were lowercases so i thought that was a convention.

Comment (ikemHood) [reply src/list.json:140810]: looked again and it seems like a convention.

Comment (Luluno01) [reply src/list.json:140810]: Thanks a lot! It is not a convention; it is just that I generated the initial values from the domain names. Most of those abbreviations are initialized with generated/inferred values. That is why most of them are lower cased. Humans overwrites are welcomed over best-guess inference.

**Comment by Luluno01** [comment]
Thanks for the update! It will take some time to deploy the updates.

**Comment by ikemHood** [comment]
> Thanks for the update! It will take some time to deploy the updates.
Glad to make them. :)

**Pull request merged**